services/frontend/src/components/ProjectCard/ProjectCard.js

```diff
@@ -36,11 +37,12 @@ const ProjectCard = ({className, name, description, skills, isActive}) => {
                         </div>
                     )}
                 </div>
-
                 <div className="project-card-nav-section">
-                    <IconButton>
-                        <ChevronRight className="project-card-nav" />
-                    </IconButton>
+                    <Link to={"/app/projects/"+id}>
```

Comment:
I prefer using template strings instead of concatenations for things like, just cause it looks cleaner.
i.e.
```suggestion
                    <Link to={`/app/projects/${id}`}>
```